rsmtool/configuration_parser.py

```diff
@@ -1051,19 +876,7 @@ def normalize_config(cls, config):
 
         for field_name in config:
 
-            if field_name in FIELD_NAME_MAPPING:
-                norm_field_name = FIELD_NAME_MAPPING[field_name]
-                warnings.warn("""The field name "{}" is deprecated """
-                              """and will be removed  in a future """
-                              """release, please use the """
-                              """new field name "{}" """
-                              """instead.""".format(field_name,
-                                                    norm_field_name),
-                              category=DeprecationWarning)
-            else:
-                norm_field_name = field_name
-
-            new_config[norm_field_name] = config[field_name]
+            new_config[field_name] = config[field_name]
 
         # Convert old values for prediction scaling:
         if 'use_scaled_predictions' in new_config:
```

Comment:
Can we also deprecate the old values for scaling predictions: "scale" and "raw"?